tests/test_provider.py

```diff
@@ -154,13 +160,15 @@ def get_path(self, oid):
     assert got_file_event
 
 
+shared_folder_test_namespaces = {
+    "onedrive": Namespace("shared", "shared|dd7f222f46bac64|DD7F222F46BAC64!2041"),
```

Comment:
Are these ids specific to a particular account?

Reply:
they are, probably would better as env vars

Reply:
Agreed, and if you are familiar with the process of getting the namespace id from onedrive, can you add that here as well?

Reply:
done.